runtime/storage_test.go

```diff
@@ -466,7 +466,7 @@ func TestRuntimePublicCapabilityBorrowTypeConfusion(t *testing.T) {
 
         // Create a public capability to the stored Vault that only exposes
         // the balance field through the Balance interface
-        self.account.link<&DapperUtilityCoin.Vault{FungibleToken.Balance}>(
+        self.account.link<&DapperUtilityCoin.Vault>(
```

Comment:
Here and below, I think it might be better to keep the interface set here
```suggestion
        self.account.link<&{FungibleToken.Balance}>(
```

Reply:
See previous comment, this is no longer the suggested best practice post-entitlements.